### PR TITLE
Improving README

### PR DESCRIPTION
**Pull request by lazyfrosch**
Some formatting, typos, Puppet style

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,31 +20,36 @@
...
 ## Module Description
-This module installs and configures Icinga 2 on your Linux or Windows hosts. By default it will use packages provided by
-your distributions repository, respectively chocolatey on Windows. The module can be enabled to use [packages.icinga.com]
-as primary repository, which will give you the ability to install the latest Icinga 2 release. All features and objects
-available in Icinga 2 can be enabled and configured with this module.
+
+This module installs and configures Icinga 2 on your Linux or Windows hosts
+ 
+By default it will use packages provided by your distributions repository, respectively
+[Chocolatey] on Windows.
```
Comment (dnsmichi) [inline README.md:33]: There's a link missing.

File: README.md [hunk 2]
```diff
@@ -20,31 +20,36 @@
...
 
 >> This Module is a rewrite of [puppet-icinga2] and will replace it in the future.
 
 ## Module Description
-This module installs and configures Icinga 2 on your Linux or Windows hosts. By default it will use packages provided by
-your distributions repository, respectively chocolatey on Windows. The module can be enabled to use [packages.icinga.com]
-as primary repository, which will give you the ability to install the latest Icinga 2 release. All features and objects
-available in Icinga 2 can be enabled and configured with this module.
+
+This module installs and configures Icinga 2 on your Linux or Windows hosts
```
Comment (dnsmichi) [inline README.md:30]: Missing dot at the end.

File: README.md [hunk 3]
```diff
@@ -158,140 +178,150 @@ class{ 'icinga2::feature::idopgsql':
...
 ### Clustering Icinga 2
+
 Icinga 2 can run in three different roles:
 
-* A master node which is on top of the hierarchy.
-* A satellite node which is a child of a satellite or master node.
-* A client node which works as an agent connected to master and/or satellite nodes.
+* in a master zone which is on top of the hierarchy
+* in a satellite zone which is a child of a satellite or master zone
+* a standalone client node/zone which works as an agent connected to master and/or satellite zones
```
Comment (dnsmichi) [inline README.md:186]: First character must be capitalised, line ends with dots.

File: README.md [hunk 4]
```diff
@@ -158,140 +178,150 @@ class{ 'icinga2::feature::idopgsql':
...
+
+A Master zone has no parent and is usually also the place where you enable the IDO and notification features. A master
+sends configurations over the Icinga 2 protocol to satellites and/or clients.
 
 More detailed examples can be found in the [examples] directory.
 
-Ths examples creates the configuration for a Master that has one Satellite connected. A global zone is created for
-templates and all features of a typical master are enabled.
+Ths examples creates the configuration for a master that has one satellite connected. A global zone is created for
+templates, and all features of a typical master are enabled.
```
Comment (dnsmichi) [inline README.md:200]: "This example creates".

File: README.md (hunk 1 above)
Comment (lazyfrosch) [reply README.md:33]: No, linked on bottom :)

File: README.md (hunk 2 above)
Comment (lazyfrosch) [reply README.md:30]: You can update the branch yourself anytime :)

**Comment by lazyfrosch** [comment]
You can update the branch yourself anytime :)

**Pull request merged**